### PR TITLE
dojson: ignore Open Access redundant license

### PR DESCRIPTION
**Pull request by kaplun**
Sentry: https://sentry.cern.ch/inspire-sentry/inspire-labs-qa/group/601709/

### CODE REVIEW

File: inspirehep/dojson/hep/fields/bd5xx.py [hunk 1]
```diff
@@ -209,8 +209,12 @@ def funding_info2marc(self, key, value):
 @utils.filter_values
 def license(self, key, value):
     """Add Terms Governing Use and Reproduction Note."""
+    license_value = force_force_list(value.get('a'))
+    # We strip away redundant 'Open Access' string
+    license_value = [val for val in license_value if license_value != 'Open Access']
+    license_value = force_single_element(license_value)
```
Comment (jacquerie) [inline inspirehep/dojson/hep/fields/bd5xx.py]: This discards data. Are we modeling it correctly? Software things can be dual-licensed, why not papers?

Comment (kaplun) [inline inspirehep/dojson/hep/fields/bd5xx.py]: I have checked. Only 17 records have these Open Access license (which is not a valid name for a license) and have also a valid CC-BY-* one. These are also the very same records having more than one license.

All the other papers have valid CC-* licenses.

**Comment by david-caro** [comment]
dammit... I was working on this :)

**Comment by kaplun** [comment]
There were no comment on Sentry... Oops!

**Pull request merged**